### DISCUSSION TITLE
Legal Info - Employment Verification Failed

### DISCUSSION THREAD

**Discussion by kshmir** [microsoft/MicrosoftEdge-Extensions#184] [Discussions]
<img width="921" alt="image" src="https://github.com/user-attachments/assets/9318e6af-c5c4-40e0-b4b2-29f116ecdc7f">

I have a LLC that has a extension that runs perfectly in the Chrome Store, thought of publishing here...

I get automatically rejected and there is no clear reason or way to appeal, as soon as I click "fix", it reloads, restarts, and ends up in the same place.

**Comment by ManikanthMSFT** [comment]
Hey @kshmir, could you please share your seller ID so that we can verify and assist you further.

**Reply by kshmir** [reply]
Hey! The seller ID is 88823990

Thanks!

**Comment by ManikanthMSFT** [comment]
Hi @kshmir, we just got an update from our team, and they request additional information to proceed with your account. Acceptable document types include:

- Formation documents, such as articles of incorporation or partnership deed
- Franchise or agency appointment letters
- Government issued letters, licenses, registrations, or certificates
- Lease or tenancy documents
- Letters or statements from a financial institution or a utility company
- Records on a Government registry website (site/link must be displayed)
- Stock exchange filings or tax filing records
- Certificate of incorporation or registration
- Extract from commercial register
- Business license
- Tax certificate
- DUNS certification
Please ensure that the documents submitted are issued within the previous 12 months or have an expiration date that is at least 2 months away.

You can either upload the required document or contact us at ext_dev_support@microsoft.com for further assistance.

Thank you for your cooperation.

**Comment by kshmir** [comment]
Hey! I uploaded the document! Still didn't get any answer!

**Comment by ManikanthMSFT** [comment]
Hi @kshmir,  thank you for your confirmation. We have informed our team, and we will write back to you once we have an update.

**Reply by kshmir** [reply]
Hey! We still didn't have any updates, we added our bank account info.

**Reply by kshmir** [reply]
Hey, any news @ManikanthMSFT ? We have over 3 weeks with this.

**Reply by kshmir** [reply]
@ManikanthMSFT we've contacted you over every mean possible for over a month. We have clients requesting this feature and we want to bring the best possible security and privacy to our customers, does Microsoft not care about this?

**Reply by kshmir** [reply]
@ManikanthMSFT @RanadhirMSFT @varaduMSFT at this point we don't really know what to do, we have contacted y'all over multiple ways, and we're still stuck!

**Comment by morozRed** [comment]
Same problem here, we're getting automatically rejected but we were never asked to provide any documents, so I'm having a hard time to understand where should we upload them (there's no "Upload" button or anything) :)

Our seller_id: 88860880

**Reply by ManikanthMSFT** [reply]
Hey @morozRed, sorry for the inconvenience caused. Could you please provide us with a screenshot of the error? This will help us assist you better.

**Comment by kshmir** [comment]
Bumping this up, sadly still not resolved, I'm very disappointed at Microsoft with the support for this.

**Reply by ManikanthMSFT** [reply]
Hey @kshmir, I apologize for the inconvenience and understand your frustration. Our team is actively working on resolving this issue. We will provide you with an update as soon as we have more information.

Thank you for your patience.

**Comment by ManikanthMSFT** [comment]
Hi @kshmir @morozRed , we appreciate your patience. We have received an update from our team advising account holders to create an SR ticket using the [Partner Center](https://partner.microsoft.com/en-us/support?stage=1). This will route you to the appropriate team best suited to address your concerns.
